### PR TITLE
Add component

### PR DESCRIPTION
**Pull request by Wes-Isaac**
# Math magicians: components

In this pull request I did the following:

- Delete all the boilerplate from CRA.
- Create a directory called components.
- inside components create a new Calculator.js file.
- In Calculator.js, create a React component for the calculator.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,17 +1,11 @@
...
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    {/* <App /> */}
```
Comment (rukundoeric) [inline src/index.js]: - [x] <b>REQUIRED:</b> To make sure that your codebase is always clean, remember to remove commented-codes.

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,11 @@
...
+    <Calculator />
   </React.StrictMode>,
   document.getElementById('root'),
 );
```
Comment (rukundoeric) [inline src/index.js]: - [x] <b>REQUIRED:</b> `App` component should be created in `src/component` directory as requested by the project. and it should be the one imported and used here, not `Calculator`.  `Calculator` should be imported and used in `App` component instead. Kindly fix it.

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @Wes-Isaac, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @Wes-Isaac, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**